l10n_eu_nace/tests/test_res_partner_industry_eu_nace_wizard.py

```diff
@@ -0,0 +1,151 @@
+# Copyright 2024 Moduon Team S.L. <info@moduon.team>
+# License LGPL-3.0 or later (https://www.gnu.org/licenses/lgpl).
+from unittest.mock import patch
+
+from odoo.tests.common import Form, TransactionCase, new_test_user, users
+
+from .test_en_nace_request_results import (
+    NACE_COMMON,
+    NACE_EN,
+    NACE_ES,
+    NACE_FR,
+    create_response,
+)
+
+MOCK_PATH = (
+    "odoo.addons.l10n_eu_nace.wizard.res_partner_industry_eu_nace_wizard.requests.get"
+)
+
+
+class TestResPartnerIndustryEUNaceWizard(TransactionCase):
+    @classmethod
+    def setUpClass(cls):
+        super().setUpClass()
+        cls.user = new_test_user(
+            cls.env,
+            login="test-manager",
+            groups="base.group_system",
+        )
+
+    def activate_langs(self, lang_list):
+        lang_ids = self.env["res.lang"]
+        lang_ids.search([("code", "not in", ["en_US"])]).write({"active": False})
+        for lang in lang_list:
+            lang_ids |= self.env.ref(lang)
+        lang_ids.write({"active": True})
+
+    def wizard_eu_nace(self):
+        wizard = Form(
+            self.env["res.partner.industry.eu.nace.wizard"].with_user(self.user)
+        )
+        import_wizard = wizard.save()
+        return import_wizard
+
+    @users("test-manager")
```

Comment:
suggestion: You can remove this decorator, if you follow my suggestion above in `setUpClass`. The same applies for all other uses of `@users`

Reply:
Done!